doc/samples/keepalived.conf.misc_check

```diff
@@ -21,7 +21,7 @@ virtual_server 10.10.10.2 1358 {
         weight 1
         MISC_CHECK {
           misc_path /usr/local/bin/script.sh
-!          misc_dynamic
+          ! misc_dynamic
```

Comment:
It is easier to see the comment mark if it as the start of the line which is why it is there rather than close to the misc_dynamic.